src/sagemaker/lineage/query.py

```diff
@@ -230,7 +230,31 @@ class PyvisVisualizer(object):
     """Create object used for visualizing graph using Pyvis library."""
```

Comment:
Is it made clear anywhere that the user needs to install Pyvis (like https://w.amazon.com/index.php/Coral/Model/XML/Traits#Pagination says for Sklearn)?

Reply:
Requirement added to visualize() documentation